### ISSUE TITLE
[A11y] Explore Collections - Grid - Status messages

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1886]
### Operating System

Windows

### Browser

Edge

### Affected page

Explore Collections

### Admin user?

Yes

### Issue Description

When there are no results, the status message should probably be reflected to some degree in the title.  This probably isn't a hard failure since it's not really an error condition per se.

When a search has been performed, the title of the page should probably say something about that instead of "All Explore Collection".  Maybe like "Search Results | All Explore Collections | Oregon Digital"?

### Suggested Fix

Better titles if this is deemed worthy.  I consider this low priority but I've been wrong before....

### Can anybody QA or does this need an expert?

Anybody

### QA Steps

Make sure titles reflect the state of the search at least in a general sense
